src/features/canvass/components/LocationDialog/pages/HouseholdsPage.tsx

```diff
@@ -104,8 +162,34 @@ const HouseholdsPage: FC<Props> = ({
                   onClick={() => {
                     onSelectHousehold(household.id);
                   }}
+                  sx={{ px: 0 }}
                 >
-                  <Box flexGrow={1}>
+                  <Box alignItems="center" display="flex" flexGrow={1}>
+                    <Checkbox
+                      checked={selectedHouseholds.includes(household.id)}
+                      onClick={(e) => {
+                        e.stopPropagation();
+                        setReportMany(true);
```

Comment:
Correct me if I'm wrong, but I believe that the `reportMany` state should be `true` whenever there is at least one selected household, and you have achieved this by setting it to true every time a household is selected. But why not just use `const reportMany = selectedHouseholds.length > 0`?